### PR TITLE
chore(golang): add rsync to golang image

### PR DESCRIPTION
**Pull request by catsoap**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -129,6 +129,7 @@ golang:
         - mockgen -version
         - modd --version
         - swagger version
+        - rsync --version
```
Comment (mremi) [inline config.yml:132]: please sort by command name :)

File: golang/Dockerfile.j2 [hunk 2]
```diff
@@ -15,7 +15,7 @@ RUN echo "Starting...\n" && \
 
   echo "Installing AWS CLI..." && \
   apt-get update -q && \
-  apt-get -qq -y install python-pip groff-base && \
+  apt-get -qq -y install python-pip groff-base rsync && \
```
Comment (mremi) [inline golang/Dockerfile.j2:18]: same here, please sort

File: config.yml (hunk 1 above)
Comment (catsoap) [reply config.yml:131]: thanks, fixed

File: golang/Dockerfile.j2 (hunk 2 above)
Comment (catsoap) [reply golang/Dockerfile.j2]: fixed !

**Pull request merged**